packages/jest-environment-jsdom/src/index.js

```diff
@@ -22,14 +23,17 @@ class JSDOMEnvironment {
   errorEventListener: ?Function;
   moduleMocker: ?ModuleMocker;
 
-  constructor(config: ProjectConfig) {
+  constructor(config: ProjectConfig, options: EnvironmentOptions = {}) {
```

Comment:
Small nit: `options?: EnvironmentOptions`.

Reply:
Good point. Actually, I'm curious to know why it's necessary to add type information to the constructor. Isn't it already defined in types/Environment.js?

Reply:
I'm not so sure, but I guess it's for the ease of writing your own custom envs and updating `flow-typed` defs, as Jest doesn't expose flow types generated from its source. 
But you're right that we should strive for easier type management; this is not ideal.

Reply:
Happy to make a PR to add the `?` :+1: